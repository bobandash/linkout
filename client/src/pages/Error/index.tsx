import { useNavigate } from 'react-router';
const ErrorPage = () => {
  const navigate = useNavigate();
  function navigateHome() {
    navigate('/');
  }
  return (
    <div className="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white">
      <div className="mx-14 flex flex-col items-center justify-center gap-3">
        <h1 className="text-9xl font-bold">404</h1>
        <div className="text-center">
          <p className="text-3xl text-slate-100">Page Not Found</p>
          <p>The page that you are searching does not exist.</p>
        </div>
        <button
          onClick={navigateHome}
          className="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
        >
          Homepage
        </button>
      </div>
    </div>
  );
};

export default ErrorPage;
