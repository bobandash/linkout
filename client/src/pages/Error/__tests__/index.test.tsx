import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Error from '../index';
import { MemoryRouter } from 'react-router';

describe('Error Page', () => {
  it('should render 404 page by default', () => {
    const container = render(
      <MemoryRouter>
        <Error />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });

  it('should render 401 page when 401 status is passed', () => {
    const container = render(
      <MemoryRouter>
        <Error errorCode={401} />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });

  it('should render 500 page when 500 status is passed', () => {
    const container = render(
      <MemoryRouter>
        <Error errorCode={500} />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });

  it('should render 404 page when 404 status is passed', () => {
    const container = render(
      <MemoryRouter>
        <Error errorCode={404} />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
