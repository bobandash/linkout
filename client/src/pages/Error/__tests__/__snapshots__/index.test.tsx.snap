// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Error Page > should render 401 page when 401 status is passde 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
      >
        <div
          class="mx-14 flex flex-col items-center justify-center gap-3"
        >
          <h1
            class="text-9xl font-bold"
          >
            401
          </h1>
          <div
            class="text-center"
          >
            <p
              class="text-3xl text-slate-100"
            >
              Unauthorized
            </p>
            <p>
              You are not authorized to view this page.
            </p>
          </div>
          <button
            class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
          >
            Homepage
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
    >
      <div
        class="mx-14 flex flex-col items-center justify-center gap-3"
      >
        <h1
          class="text-9xl font-bold"
        >
          401
        </h1>
        <div
          class="text-center"
        >
          <p
            class="text-3xl text-slate-100"
          >
            Unauthorized
          </p>
          <p>
            You are not authorized to view this page.
          </p>
        </div>
        <button
          class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
        >
          Homepage
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Error Page > should render 401 page when 401 status is passed 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
      >
        <div
          class="mx-14 flex flex-col items-center justify-center gap-3"
        >
          <h1
            class="text-9xl font-bold"
          >
            401
          </h1>
          <div
            class="text-center"
          >
            <p
              class="text-3xl text-slate-100"
            >
              Unauthorized
            </p>
            <p>
              You are not authorized to view this page.
            </p>
          </div>
          <button
            class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
          >
            Homepage
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
    >
      <div
        class="mx-14 flex flex-col items-center justify-center gap-3"
      >
        <h1
          class="text-9xl font-bold"
        >
          401
        </h1>
        <div
          class="text-center"
        >
          <p
            class="text-3xl text-slate-100"
          >
            Unauthorized
          </p>
          <p>
            You are not authorized to view this page.
          </p>
        </div>
        <button
          class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
        >
          Homepage
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Error Page > should render 404 page by default 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
      >
        <div
          class="mx-14 flex flex-col items-center justify-center gap-3"
        >
          <h1
            class="text-9xl font-bold"
          >
            404
          </h1>
          <div
            class="text-center"
          >
            <p
              class="text-3xl text-slate-100"
            >
              Page Not Found
            </p>
            <p>
              The page that you are searching does not exist.
            </p>
          </div>
          <button
            class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
          >
            Homepage
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
    >
      <div
        class="mx-14 flex flex-col items-center justify-center gap-3"
      >
        <h1
          class="text-9xl font-bold"
        >
          404
        </h1>
        <div
          class="text-center"
        >
          <p
            class="text-3xl text-slate-100"
          >
            Page Not Found
          </p>
          <p>
            The page that you are searching does not exist.
          </p>
        </div>
        <button
          class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
        >
          Homepage
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Error Page > should render 404 page when 401 status is passde 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
      >
        <div
          class="mx-14 flex flex-col items-center justify-center gap-3"
        >
          <h1
            class="text-9xl font-bold"
          >
            404
          </h1>
          <div
            class="text-center"
          >
            <p
              class="text-3xl text-slate-100"
            >
              Page Not Found
            </p>
            <p>
              The page that you are searching does not exist.
            </p>
          </div>
          <button
            class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
          >
            Homepage
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
    >
      <div
        class="mx-14 flex flex-col items-center justify-center gap-3"
      >
        <h1
          class="text-9xl font-bold"
        >
          404
        </h1>
        <div
          class="text-center"
        >
          <p
            class="text-3xl text-slate-100"
          >
            Page Not Found
          </p>
          <p>
            The page that you are searching does not exist.
          </p>
        </div>
        <button
          class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
        >
          Homepage
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Error Page > should render 404 page when 404 status is passed 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
      >
        <div
          class="mx-14 flex flex-col items-center justify-center gap-3"
        >
          <h1
            class="text-9xl font-bold"
          >
            404
          </h1>
          <div
            class="text-center"
          >
            <p
              class="text-3xl text-slate-100"
            >
              Page Not Found
            </p>
            <p>
              The page that you are searching does not exist.
            </p>
          </div>
          <button
            class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
          >
            Homepage
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
    >
      <div
        class="mx-14 flex flex-col items-center justify-center gap-3"
      >
        <h1
          class="text-9xl font-bold"
        >
          404
        </h1>
        <div
          class="text-center"
        >
          <p
            class="text-3xl text-slate-100"
          >
            Page Not Found
          </p>
          <p>
            The page that you are searching does not exist.
          </p>
        </div>
        <button
          class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
        >
          Homepage
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Error Page > should render 500 page when 500 status is passde 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
      >
        <div
          class="mx-14 flex flex-col items-center justify-center gap-3"
        >
          <h1
            class="text-9xl font-bold"
          >
            500
          </h1>
          <div
            class="text-center"
          >
            <p
              class="text-3xl text-slate-100"
            >
              Internal Server Error
            </p>
            <p>
              There was an internal server error. Please try again later.
            </p>
          </div>
          <button
            class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
          >
            Homepage
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
    >
      <div
        class="mx-14 flex flex-col items-center justify-center gap-3"
      >
        <h1
          class="text-9xl font-bold"
        >
          500
        </h1>
        <div
          class="text-center"
        >
          <p
            class="text-3xl text-slate-100"
          >
            Internal Server Error
          </p>
          <p>
            There was an internal server error. Please try again later.
          </p>
        </div>
        <button
          class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
        >
          Homepage
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Error Page > should render 500 page when 500 status is passed 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
      >
        <div
          class="mx-14 flex flex-col items-center justify-center gap-3"
        >
          <h1
            class="text-9xl font-bold"
          >
            500
          </h1>
          <div
            class="text-center"
          >
            <p
              class="text-3xl text-slate-100"
            >
              Internal Server Error
            </p>
            <p>
              There was an internal server error. Please try again later.
            </p>
          </div>
          <button
            class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
          >
            Homepage
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex h-screen w-full flex-col items-center justify-center gap-5 bg-color_1 text-white"
    >
      <div
        class="mx-14 flex flex-col items-center justify-center gap-3"
      >
        <h1
          class="text-9xl font-bold"
        >
          500
        </h1>
        <div
          class="text-center"
        >
          <p
            class="text-3xl text-slate-100"
          >
            Internal Server Error
          </p>
          <p>
            There was an internal server error. Please try again later.
          </p>
        </div>
        <button
          class="w-full border-2 border-white bg-color_3 p-3 text-2xl uppercase"
        >
          Homepage
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
